docs/examples/download_file.py

```diff
@@ -5,7 +5,7 @@
 
 def download_file(reactor, url, destination_filename):
     destination = open(destination_filename, 'wb')
-    d = treq.get(url)
+    d = treq.get(url, unbuffered=True)
```

Comment:
Strange, I [made this change myself some time ago](https://github.com/twm/treq/commit/d4c772566be2c6a4909dee0161b9af607ebfbd1c). It must have gotten clobbered in a merge.